styles.css

```diff
@@ -1 +1,63 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.grid-container {
+    background-color: hsl(25, 18%, 44%);
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+            "header"
+            "content-b"
+            "content-c"
+            "aside-d"
+            "aside-e"
+            "."
+            "footer";
+}
+
+.grid-item {
+    background-color: hsl(327, 13%, 73%);
+    font-family: "Avenir Next", sans-serif;
+    padding: 8px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-header {
+    grid-area: header;
+}
+
+.grid-content-b {
+    grid-area: content-b;
+}
+
+.grid-content-c {
+    grid-area: content-c;
+}
+
+.grid-aside-d {
+    grid-area: aside-d;
+}
+
+.grid-aside-e {
+    grid-area: aside-e;
+}
+
+.grid-footer {
+    grid-area: footer;
+}
+
```

Comment:
Zelf heb ik hier ook gebruik gemaakt van classes terwijl in de uitwerkingen de items als id zijn gemaakt.